RecoLocalCalo/HcalRecProducers/python/HBHEPhase1Reconstructor_cfi.py

```diff
@@ -69,7 +71,10 @@
         useM2 = cms.bool(True),
 
         # Use "Method 3"?
-        useM3 = cms.bool(True)
+        useM3 = cms.bool(True),
+
+        # Use Mahi?
+        useMahi = cms.bool(True)
```

Comment:
useM2 is still set to True.
Please turn it off and perhaps check at parsing step that only one is enabled